src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -188,7 +188,15 @@ public override void OnStopActivity(Activity activity, object payload)
 
                 var response = context.Response;
 
-                activity.SetTag(SemanticConventions.AttributeHttpStatusCode, response.StatusCode);
+                //activity.SetTag(SemanticConventions.AttributeHttpStatusCode, response.StatusCode);
+                if (response.StatusCode > 200 && response.StatusCode <= 299)
+                {
+                    activity.SetStatus(ActivityStatusCode.Ok);
```

Comment:
dont set OK. Only set status, if Error. Else leave it. 
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status

"Generally, Instrumentation Libraries SHOULD NOT set the status code to Ok, unless explicitly configured to do so. Instrumentation Libraries SHOULD leave the status code as Unset unless there is an error, as described above."